configuration/external-storage.md

```diff
@@ -68,9 +68,11 @@ You can modify the `fstab` file to define the location where the storage device
 4. Add the following line in the `fstab` file:
 
     ```
-    UUID=5C24-1453 /mnt/mydisk FSTYPE defaults,auto,umask=000,users,rw,nofail 0 0
+    UUID=5C24-1453 /mnt/mydisk FSTYPE defaults,auto,users,rw,nofail 0 0
     ```
    Replace FSTYPE with the type of your file system, which you found in step 2 of 'Mounting a storage device' above.
+   
+5. If the filesystem type is FAT or NTFS, add the option `umask=000` immediately after `nofail` - this will allow all users full read/write access to every file on the storage device.
 
```

Comment:
I think having the complete line with the umask bit inserted here would make this clearer - at the moment do you need a comma after nofail for example?

Reply:
Oops - forgot about the comma. The current wording matches how the page already suggests the `x-systemd.device-timeout` option for something else.